classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -1047,7 +1049,7 @@ protected function writeImageOnDisk($base_path, $new_path, $dest_width = null, $
      * @param array $image_types
      * @param string $parent_path
      *
-     * @return bool|void
+     * @return bool|string|void
```

Comment:
I maybe missing something but I don't think that this function can return `void` 🤔

Reply:
I agree, void is never returned all the edge cases throw exceptions